ci-operator/step-registry/fips-check/optional-operator-images-scan/fips-check-optional-operator-images-scan-commands.sh

```diff
@@ -130,17 +130,23 @@ for package in ${package_list}; do
         csv_test_result="${data_dir}/${package}/$csv"
         echo "##Starting scan for CSV '$csv'" >> $csv_test_result
         # retrieve related images from current CSV
-        image_list=$(cat /tmp/$catalog.json | jq -r '.items[].status.channels[]|select(.currentCSV=="'$csv'")|.currentCSVDesc.relatedImages[]' || true);
+        image_list_string=$(cat /tmp/$catalog.json | jq -r '.items[].status.channels[]|select(.currentCSV=="'$csv'")|.currentCSVDesc.relatedImages[]' | sort | uniq || true);
+        image_list_string=$(echo $image_list_string)
```

Comment:
```suggestion
```
Nit: if this line can be removed?

Otherwise LGTM

Reply:
No, I have tried. The image_list length will be 1 if remove it(although for my local env it is not need).

Reply:
Ok, thanks for your confirm！Deferring the rehearsal-ack and final approval to @xingxingxia

/lgtm